HW_04-1/Task2-2/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+        }
+    }
```

Comment:
???

Reply:
Accidentally added... Task2-2 hasn't meant to exist.